content/platform/swarm-learning/home.md

```diff
@@ -0,0 +1,66 @@
+
+- - -
+
+## Articles
+
+* **[2﻿021 Nature Paper - Swarm Learning for decentralized and confidential clinical machine learning](https://www.nature.com/articles/s41586-021-03583-3)**
+* **[HPE Labs - Swarm Learning: Turn your distributed data into competitive edge](https://www.hpe.com/psnow/doc/a50000344enw?jumpid=in_lit-psnow-red)**
+* **[Research gate - Swarm Learning as a privacy-preserving machine learning approach for disease classification](https://www.researchgate.net/publication/342495847_Swarm_Learning_as_a_privacy-preserving_machine_learning_approach_for_disease_classification)**
+
+- - -
+
+## Learn more about HPE Swarm Learning and its underlying technology
+
+**\-﻿ What's All the Buzz about Swarm Learning?**
+
+In this video, Dr. Eng Lim Goh highlights the challenges of data privacy in the organizations to use siloed data to be used to bring out collective insights. Also, he explains HPE Swarm learning has the potential to change our lives and solve some of the world's most pressing and complex problems.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/JqwjyENd89I" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+**\-﻿ The Big Shift: What is Swarm Learning?**
+
+Swarm learning is the next gold rush for machine intelligence - training at the edge so the edge devices get smarter and also train their peers. With no central authority, Blockchain is integrated to add control, privacy, and security.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/6Fep6Lw5t-U" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+**\-﻿ HPE Swarm Learning Product Introduction**
+
+HPE Swarm Learning – industry’s first privacy preserving, decentralized machine learning solution . Developed by HPE Labs, HPE Swarm Learning enables AI/ML model developers to increase accuracy and reduces biases in their models.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/ORujFJ1lVVw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+- - -
+
+## Listen to what the Experts are saying
+
+\-﻿ **How to Optimize Machine Learning at the Edge with Swarm Learning**
+
+Ronald van Loon and Krishnaprasad Shastry, Chief Technologist at HPE, talked about optimizing Machine Learning at the edge with Swarm Learning.
```

Comment:
You may want to add "In this video," prior to "Ronald van Loon". In addition, if you are including Krishna's job title, you should either also include Ronald's OR you can write it like this:

In this video, Ronald van Loon and HPE Chief Technologist, Krishnaprasad Shastry, talk about...

Note that you'll probably want to change "talked" to "talk" and in the rest of this sentence "machine learning" and "swarm learning" should use lower case initial letters.

Reply:
done